core/templates/services/prevent-page-unload-event.service.ts

```diff
@@ -25,35 +25,46 @@ import { WindowRef } from 'services/contextual/window-ref.service';
 })
 export class PreventPageUnloadEventService {
   private listenerActive: boolean;
+  validation: Function;
```

Comment:
Can we call this validationCallback or similar so that it is clear that it is a function callback?

Current name does not accurately reflect that it is a function (note functions usually start with a verb but in this case, we are using the reference to a function so it becomes a bit tricky to name).